### PR TITLE
If description or summary are left empty, workspace request fails

### PR DESCRIPTION
**Pull request by shafi1379**
@afoerster @rk-fsdev @satish-sanjeev

### CODE REVIEW

**Comment by rk-fsdev** [comment]
@shafi1379 , could you check why some build checkes were not successful?

**Comment by rk-fsdev** [comment]
@shafi1379 , and in workspace request page, we need to set `description` and `summary` inputs fields as required.

File: ui/containers/WorkspaceRequest/components/OverviewPage.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default class OverviewPage extends React.Component<Props> {
     const { request, setRequest } = this.props;
     setRequest({
       ...request,
-      [key]: e.target.value,
+      [key]: e.target.value.trim().length ? e.target.value : '',
```
Comment (afoerster) [inline ui/containers/WorkspaceRequest/components/OverviewPage.tsx:17]: what's the behavior here Shafi? Is an error thrown? The user should be required to fill in these fields before continuing.

Comment (shafi1379) [reply ui/containers/WorkspaceRequest/components/OverviewPage.tsx:17]: It won't allow user until type at least one character. Previously it was allowing space

**Comment by shafi1379** [comment]
> @shafi1379 , and in workspace request page, we need to set `description` and `summary` inputs fields as required.

Now, it won't allow user without entering at least one character. do we need to highlight with red border?

**Comment by afoerster** [comment]
Yeah we should, otherwise they might be confused why they can't move
forward.

**Comment by shafi1379** [comment]
Highlight will get enabled either user empty the field or touching the field.

**Comment by rk-fsdev** [comment]
@shafi1379 , I am sure your solution works properly. Could you dig into it by using `getFieldDecorator` of antd form?
```
        <Form.Item validateStatus={passwordError ? 'error' : ''} help={passwordError || ''}>
          {getFieldDecorator('password', {
            rules: [{ required: true, message: 'Please input your Password!' }],
          })(
            <Input
              prefix={<Icon type="lock" style={{ color: 'rgba(0,0,0,.25)' }} />}
              type="password"
              placeholder="Password"
            />,
          )}
        </Form.Item>
```
I think if you use `getFieldDecorator` of antd, we don't need `isDescriptionTouched` states.

**Review by rk-fsdev — APPROVED** [review]

**Pull request merged**